docs/ha.md

```diff
@@ -74,6 +74,23 @@ sudo ovs-appctl -t /var/run/openvswitch/ovnsb_db.ctl \
     cluster/status OVN_Southbound
 ```
 
+## Start 'ovn-kube -init-cluster-manager'
+
+On any one of the masters or on all the masters, we need to start
+'ovnkube -init-cluster-manager'.  If it is started on only one master,
+then this should ideally be a daemonset with replica count of 1.
+If it is started on all the masters (for HA of --init-cluster-manager),
+then this should ideally be a deployment.
+
```

Comment:
Which option would you recommend for cluster manager? A deployment with leader election across all master nodes, just like we do today for ovnk master, or one single instance on one master node?